ld/nodemcu.ld

```diff
@@ -103,7 +103,11 @@ SECTIONS
     *(.init.literal)
     *(.init)
     
-    *sdk/esp_iot_sdk_*lib/lib*.a:*(.literal .text)
+    *libmain.a:*(.literal .text)
+    *libnet80211.a:*(.literal .text)
+    *libphy.a:*(.literal .text)
+    *libpp.a:*(.literal .text)
+    *libgcc.a:*(.literal .text)
```

Comment:
Hah! Well I can't argue too strongly here seeing as I've used the "I've reviewed and tested it, and it works" argument myself in regard to flash vs ram.

Could we perhaps leave a comment in the `nodemcu.ld` file indicating that we're specifically excluding `ilbwps.a` due to size constraints?